tox.ini

```diff
@@ -7,7 +7,7 @@ deps =
     django42: Django>=4.2,<5.0
 
 commands =
-    pytest
+    pytest {posargs}
```

Comment:
this is not related to the PR, it just makes it easier to run test for a single file or object locally